setup.py

```diff
@@ -56,7 +56,9 @@
     ],
     python_requires=">=3.6",
     # use `pip install data-science-types[dev]` to install development packages
-    extras_require={"dev": ["black", "mypy", "flake8", "flake8-pyi"]},
+    extras_require={
```

Comment:
btw, did a new environment to run the 36 tests locally. Couldn't run without installing these extra packages for the tests, so added them